src/bedrock/api.js

```diff
@@ -11,13 +11,75 @@ module.exports = class BedrockRealmAPI extends RealmAPI {
     return { host, port: Number(port) }
   }
 
-  async getRealmFromInvite (realmInviteCode) {
+  async getRealmFromInvite (realmInviteCode, skipAccept = false) {
```

Comment:
You can also set this parameter to be invite=true. I think this would be clearer.